### PR TITLE
Updates from IESG reviews/comments/discusses

### PR DESCRIPTION
**Pull request by louberger**

### CODE REVIEW

File: ip/draft-ietf-detnet-ip.xml [hunk 1]
```diff
@@ -246,9 +246,9 @@ To be discussed:
       </t>
 	  <t>
 	    For some of the protocols 5-tuples and 6-tuples cannot be used because 
-		the port information is not available (e.g., ICMP, IPSec ESP). Same can 
-		be valid for flow aggregates. In such cases using smaller tuples are 
-		appropriate, e.g., a 3-tuple (2 IP addresses, IP protocol) or even a 
+		the port information is not available (e.g., ICMP, IPSec ESP). This is
+		also the case for flow aggregates. In such cases, using
+                few fields is appropriate, e.g., a 3-tuple (2 IP addresses, IP protocol) or even a
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml]: a few fields

File: ip/draft-ietf-detnet-ip.xml [hunk 2]
```diff
@@ -421,6 +421,10 @@ To be discussed:
 		  end systems. From a practical standpoint this
           means that all nodes along the end-to-end path of DetNet flows need
           to agree on what fields are used for flow identification.
+          Possible consequences of not having such an agreement include
+          having some flows interfering with other flows, and for a
+          the traffic treatment expected for a service not being
+          provided.
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml:450]: Possible consequences of not having such an agreement include
some flows interfering with other flows, and the traffic 
treatment expected for a service not being
provided.

File: ip/draft-ietf-detnet-ip.xml [hunk 3]
```diff
@@ -1000,8 +1005,9 @@ DetNet                    |L2/SbN|          |L2/SbN|
             configuration, e.g., via the controller or management plane.
           </t>
           <t>
-            Information identifying a DetNet flow is ordered and
-            implementations use the first match.  This can, for example, be
+            An implementation MUST support ordering of the
+            set of information information used to identify an
+            individual DetNet flow.  This can, for example, be
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml:1034]: An implementation MUST support using a set of information to identify an individual flow to support ordering for that flow.

File: ip/draft-ietf-detnet-ip.xml [hunk 4]
```diff
@@ -627,7 +630,7 @@ DetNet                    |L2/SbN|          |L2/SbN|
           highlight the implications of DetNet IP flow identification on
           path selection and next hops.  As mentioned above, the DetNet
           IP data plane identifies flows using "6-tuple" header
-          information as well as the additional optional header
+          information as well as the additional optional (flow label) header
           field. DetNet generally allows for both flow-specific traffic
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml:653]: information as well as the optional (flow label) header

File: ip/draft-ietf-detnet-ip.xml [hunk 5]
```diff
@@ -1042,7 +1045,9 @@ DetNet                    |L2/SbN|          |L2/SbN|
         The primary consideration for the DetNet data plane is to maintain
         integrity of data and delivery of the associated DetNet service traversing
         the DetNet network.
-        The integrity, and confidentiality, of application flows can be protected through whatever means are
+        Since no DetNet-specific fields are available in the DetNet IP
+        data plane,
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml:1078]: Since no DetNet-specific fields are utilized in the DetNet IP

File: ip/draft-ietf-detnet-ip.xml [hunk 6]
```diff
@@ -274,7 +274,9 @@ To be discussed:
       <t>
         The specific procedures that are required to be implemented by a
         DetNet node supporting this document can be found in <xref
-        target="ip-procs"/>.
+        target="ip-procs"/>.  The DetNet controller plane, as defined in
+        <xref target="RFC8655"/>, is responsible for providing each node
+        with the information needed to identify an handle each DetNet flow.
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml]: needed to identify and handle each DetNet flow.

File: ip/draft-ietf-detnet-ip.xml [hunk 7]
```diff
@@ -838,7 +838,7 @@ DetNet                    |L2/SbN|          |L2/SbN|
               when processing IPv6 packets. This includes 
               the next protocol
               values defined in <xref target="nxt-proto"/> and other
-              non-zero values.
+              values, including zero.
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml]: I'm not sure I understand the text went from next-protocol values for  TCP, UDP and IPsec  but not zero  and is changed to  TCP, UDP  and IPsec and Zero?

File: ip/draft-ietf-detnet-ip.xml [hunk 8]
```diff
@@ -608,10 +608,9 @@ DetNet                    |L2/SbN|          |L2/SbN|
           support leverages the underlying network layer such as
           802.1 TSN.  The node-internal traffic control mechanisms used to deliver QoS for
           IP encapsulated DetNet flows are outside the scope of this
-          document. From an encapsulation perspective, the combination of the 6-tuple i.e.,
-          the typical 5-tuple enhanced with the DSCP and previously
-          mentioned optional field (i.e., flow label), uniquely identifies a DetNet
-          IP flow.
+          document. From an encapsulation perspective, the combination of the 6-tuple
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml:611]: From an IP encapsulation perspective

File: ip/draft-ietf-detnet-ip.xml [hunk 9]
```diff
@@ -699,8 +698,8 @@ DetNet                    |L2/SbN|          |L2/SbN|
           impacts DetNet IP data plane flow identification and resource
           allocation.  As discussed above, IP flow identification uses
           the IP "6-tuple" for flow identification.  DetNet IP flows can
-          be aggregated using any of the 6-tuple, and an additional optional 
-		  field (i.e., flow label).  The use of prefixes, wildcards,
+          be aggregated using any of the 6-tuple and optionally also by
```
Comment (dfedyk) [inline ip/draft-ietf-detnet-ip.xml]: be aggregated using any of the 6-tuple fields and optionally also by

**Comment by louberger** [comment]
@dfedyk @vb-balazs -- please approve and merge -- I've uploaded this version...

**Review by vb-balazs — APPROVED** [review]

**Pull request merged**